docs/release-notes/4126-fix-post-user.txt

```diff
@@ -0,0 +1,6 @@
+:orphan:
+
+**Bug Fixes**
+
+-  API: Fix an issue where the ``POST /users`` endpoint always returned an error instead of the
```

Comment:
any chance we could add a test for this?

Reply:
do we have any similar tests that cover an API endpoint like this? trying to find something existing to reference.

Reply:
couple options:
1. Integration tests in master: https://github.com/determined-ai/determined/blob/master/master/test/integration/api/api_checkpoints_test.go
2. can be just another python test in `e2e_tests/tests/cluster`, using bindings to hit the API.

Reply:
adding test to `e2e_tests/test/cluster`